src/test/java/by/exadel/internship/formTest/FormTest.java

```diff
@@ -0,0 +1,59 @@
+package by.exadel.internship.formTest;
+
+import by.exadel.internship.InternshipApplicationTests;
+import by.exadel.internship.dto.formDTO.FormFullDTO;
+import com.fasterxml.jackson.core.type.TypeReference;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.MediaType;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+
+public class FormTest extends InternshipApplicationTests {
+
+
+    @Test
+    public void givenFormWithFile_checkFirstName()
+            throws Exception {
+
+        MockMultipartFile file = new MockMultipartFile("file", "text.txt",
+                MediaType.MULTIPART_FORM_DATA_VALUE, "some file".getBytes());
+        MockMultipartFile form = new MockMultipartFile("form", "form",
+                MediaType.APPLICATION_JSON_VALUE,
+                ("{\"firstName\": \"testName\",\"city\": \"string\",\"country\": \"string\"," +
```

Comment:
I think it's better to move test data to separate method or class.

Reply:
this testData use only once, and only in this test